src/emqx.app.src

```diff
@@ -1,7 +1,7 @@
 {application, emqx,
  [{id, "emqx"},
   {description, "EMQ X"},
-  {vsn, "4.3.13"}, % strict semver, bump manually!
+  {vsn, "4.3.14"}, % strict semver, bump manually!
```

Comment:
NIT: please help to add a comment above to explain why this version is advancing faster than the release version.
I always had to think twice when reviewing the changes.